src/app/(proper_react)/(redesign)/MobileShell.tsx

```diff
@@ -33,9 +34,19 @@ export const MobileShell = (props: Props) => {
   const l10n = useL10n();
   const [isExpanded, setIsExpanded] = useState(false);
   const recordTelemetry = useTelemetry();
+  const pathName = usePathname();
+  const wrapperRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (pathName === "/user/dashboard") {
```

Comment:
Yeah that makes sense enough - but would be good to add that as a comment, since it's a "surprising" thing when someone encounters it in this component.